__tests__/App.js

```diff
@@ -5,6 +5,8 @@ import App from '../App';
 // Note: test renderer must be required after react-native.
 import renderer from 'react-test-renderer';
 
+jest.useFakeTimers();
```

Comment:
fixes a ReferenceError regarding trying to use an import when Jest is tearing down